pkg/controller/controller_binding.go

```diff
@@ -164,13 +173,16 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 			if err != nil {
 				s := fmt.Sprintf("Failed to unmarshal Binding parameters\n%s\n %s", binding.Spec.Parameters, err)
 				glog.Warning(s)
-				c.updateBindingCondition(
+				if err := c.updateBindingCondition(
 					binding,
 					v1alpha1.BindingConditionReady,
 					v1alpha1.ConditionFalse,
 					errorWithParameters,
 					"Error unmarshaling binding parameters. "+s,
-				)
+				); err != nil {
+					glog.Errorf("updating binding condition (%s)", err)
```

Comment:
I would prefer to include the text "Error" in the message, even though I am fully aware that the log level says it's an error.